docs/ft-config.md

```diff
@@ -0,0 +1,17 @@
+# ft-config.json
```

Comment:
How about `.flow-typed-config.json`? I'm not sure `ft` would be meaningful to most people. I don't feel strongly here, but it even took me a second to realize `ft` mean "flow-typed".

Reply:
Sounds perfect.